frontend/src/metabase/collections/components/CollectionList.styled.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
```

Comment:
nit: styled file name has singular form compared to the related component file 
CollectionList.styled.js 
Collection**s**List.jsx

Reply:
Sharp eye @alxnddr 🦅 Thanks, will fix

Reply:
That was the only tiny thing I noticed, everything else looks so nice and clean. Great job 🏅